### PR TITLE
[ci] Pin master for CI tests

### PR DESCRIPTION
**Pull request by stuartmorgan**
Updates to match the recent change in flutter/plugins, to use a pinned
version of master updated by an autoroller instead of the latest version.

This affects both the Cirrus tests and the Windows LUCI tests.

Part of https://github.com/flutter/flutter/issues/93811

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [ ] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by stuartmorgan** [comment]
Once this lands (so the target file is in place) I'll get the autoroller set up.

**Comment by stuartmorgan** [comment]
Actually, since we have a fix for the breakage we can do this in two parts. I'll send a separate PR to create the file unused.

**Comment by stuartmorgan** [comment]
This PR will now just enable using the version file. We can wait until the autoroller is running to land this.

**Comment by stuartmorgan** [comment]
The autoroller is working! Its first roll has landed :) So we're ready to move forward with this.

File: .cirrus.yml
Comment (gaaclarke) [inline .cirrus.yml:15]: nit: What's a "treeish"?

**Review by gaaclarke — APPROVED** [review]

**Pull request merged**
